pmps.ui

```diff
@@ -129,7 +129,7 @@
           <item row="4" column="1">
            <widget class="PyDMLabel" name="PyDMLabel_3">
             <property name="enabled">
-             <bool>true</bool>
```

Comment:
Opps I changed these guys to `false` - it seems like even with `export PYDM_DESIGNER_ONLINE=0` - this is still happening - but maybe not - will change these back
`